test/c/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 IF( NOT Boost_FOUND )
     find_package(Boost COMPONENTS unit_test_framework)
+    find_package(Boost COMPONENTS system)
+    find_package(Boost COMPONENTS filesystem)
```

Comment:
this should be BOOST 1.48.0 since this is only for testing and C++11 and is not the same as the rest of the codebase. Basically boost 1.48.0 is the max we can get in travis-CI so this is kind of a limitation :(